recipes/vulkan-utility-libraries/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,8 @@
+#include <vulkan/layer/vk_layer_settings.hpp>
+#include <iostream>
+
+int main() {
+    VkuLayerSettingSet layerSettingSet = VK_NULL_HANDLE;
+    vkuCreateLayerSettingSet("VK_LAYER_LUNARG_conan", nullptr, nullptr, nullptr, &layerSettingSet);
+    std::cout << VK_EXT_LAYER_SETTINGS_EXTENSION_NAME << std::endl;
```

Comment:
```suggestion
    std::cout << VK_EXT_LAYER_SETTINGS_EXTENSION_NAME << std::endl;
    return 0;
```

To avoid compiler warning about not returning a promised value.

Reply:
Well, the compiler will be at fault then. C99+ and C++ standards both explicitly allow main without a return and will return 0 in that case.

C++ standard §3.6.1.5:
> If control flows off the end of the compound-statement of `main`, the effect is equivalent to a return with operand 0.